### PR TITLE
doc: Matter: new button and LED numbering

### PR DESCRIPTION
**Pull request by wiba-nordic**
The nRF54L and nRF54H series PDKs have a new numbering system. This makes the required adjustments to Matter samples that support those boards.

### CODE REVIEW

**Review by ArekBalysNordic — APPROVED** [review]
The new approach with tables looks great :)

File: samples/matter/lock/README.rst [hunk 1]
```diff
@@ -481,37 +548,10 @@ Upgrading the device firmware
...
 
-     - ``Normal`` - The lock operates normally.
-     - ``Vacation`` - Only remote operations are enabled.
-     - ``Privacy`` - All external interactions with the lock are disabled.
-       Can only be used if the lock is in the locked state.
-       Manually unlocking the lock changes the mode to ``Normal``.
-     - ``NoRemoteLockUnlock`` - All remote operations with the lock are disabled.
-     - ``Passage`` - The lock can be operated without providing a PIN.
-       This option can be used, for example, for employees during working hours.
+Enabling scheduled timed access
```
Comment (annwoj) [inline samples/matter/lock/README.rst:553]: The last 2 testing sections (Testing switching between Thread and Wi-Fi and Bluetooth LE with Nordic UART Service) also mention DK buttons

File: samples/matter/thermostat/README.rst [hunk 2]
```diff
@@ -157,27 +157,46 @@ Factory data support
...
+
+      .. include:: /includes/matter_segger_usb.txt
+
+      NFC port with antenna attached:
+          Optionally used for obtaining the `Onboarding information`_ from the Matter accessory device to start the :ref:`commissioning procedure <matter_thermostat_sample_remote_control>`.
+
+   .. group-tab:: nRF54 DKs
+
+      LED 0:
+         .. include:: /includes/matter_sample_state_led.txt
```
Comment (annwoj) [inline samples/matter/thermostat/README.rst:181]: There seems to be discrepancy between the tab with old numbering (it doesn't mention All LEDs, and the LED 2 that corresponds to the LED 1 mentioned in the nRF54 devices) - is this intended?

File: samples/matter/thermostat/README.rst [hunk 3]
```diff
@@ -206,16 +225,33 @@ Testing basic features
...
+      #. Keep **Button 1** pressed for more than six seconds to initiate factory reset of the device.
 
-The device reboots after all its settings are erased.
+         The device reboots after all its settings are erased.
+
+   .. group-tab:: nRF52, nRF53 and nRF70 DKs
+
+      1. |connect_kit|
+      #. |connect_terminal_ANSI|
+      #. Observe that **LED0** starts flashing (short flash on).
```
Comment (annwoj) [inline samples/matter/thermostat/README.rst:247]:
```suggestion
      #. Observe that **LED 0** starts flashing (short flash on).
```

File: samples/matter/thermostat/README.rst [hunk 4]
```diff
@@ -206,16 +225,33 @@ Testing basic features
...
-   The sample starts automatically printing the simulated temperature data to the terminal with 30-second intervals.
-#. Press **Button 2** to print the most recent temperature data to the terminal.
-#. Keep the **Button 1** pressed for more than six seconds to initiate factory reset of the device.
+.. tabs::
+
+   .. group-tab:: nRF52, nRF53 and nRF70 DKs
+
+      1. |connect_kit|
+      #. |connect_terminal_ANSI|
+      #. Observe that **LED1** starts flashing (short flash on).
```
Comment (annwoj) [inline samples/matter/thermostat/README.rst:234]:
```suggestion
      #. Observe that **LED 1** starts flashing (short flash on).
```

File: samples/matter/window_covering/README.rst [hunk 5]
```diff
@@ -113,60 +113,86 @@ Factory data support
...
-LED 3 (nRF52840 DK and nRF5340 DK)
-    Indicates the tilt position of the window cover, which is represented by the brightness of the LED.
-    The brightness level ranges from ``0`` to ``255``, where the brightness level set to ``0`` (switched off LED) indicates a fully opened window cover (tilted to a horizontal position) and the brightness level set to ``255`` indicates a fully closed window cover (tilted to a vertical position).
+         Additionally, the LED starts blinking evenly (500 ms on/500 ms off) when the Identify command of the Identify cluster is received on the endpoint ``1``.
+         The command's argument can be used to specify the duration of the effect.
 
-LED 4 (nRF54L15 PDK)
-    .. note::
-       This is **LED 3** on the board revision v0.3.0.
+      LED 3:
```
Comment (annwoj) [inline samples/matter/window_covering/README.rst:130]: All throughout this sample LED 3 is referred in the same places for both nRF54DKs and other DKs, is this intended?

Comment (wiba-nordic) [reply samples/matter/window_covering/README.rst:130]: This one is intended. Functionality required by the sample is not available to all LEDs in the current nRF54L15 PDKs.

Comment (annwoj) [reply samples/matter/window_covering/README.rst:130]: got it, thanks!

File: samples/matter/lock/README.rst (hunk 1 above)
Comment (wiba-nordic) [reply samples/matter/lock/README.rst:553]: The switching build type is currently only supported for the nRF5340 DK with the nRF7002 EK, so it doesn't need text for the nRF54 DKs.

Took out the LED mention from the UART section.

File: samples/matter/thermostat/README.rst (hunk 2 above)
Comment (wiba-nordic) [reply samples/matter/thermostat/README.rst:181]: Good catch. Seems to be a copy-paste error on my part, as that information is from the Light switch sample.

File: samples/matter/light_bulb/README.rst [hunk 6]
```diff
@@ -196,34 +196,53 @@ To set up an AWS IoT instance and configure the sample, complete the following s
...
+         * Off - The light bulb is off.
+
+         Additionally, the LED starts blinking evenly (500 ms on/500 ms off) when the Identify command of the Identify cluster is received on the endpoint ``1``.
+         The command's argument can be used to specify the duration of the effect.
+
+      Button 1:
+         .. include:: /includes/matter_sample_button.txt
+
+      Button 2:
+         Changes the light bulb state to the opposite one.
```
Comment (annwoj) [inline samples/matter/light_bulb/README.rst:243]:
```suggestion
      LED 0:
         .. include:: /includes/matter_sample_state_led.txt

      LED 1:
         Shows the state of the light bulb.
         The following states are possible:

         * Solid On - The light bulb is on.
         * Off - The light bulb is off.

         Additionally, the LED starts blinking evenly (500 ms on/500 ms off) when the Identify command of the Identify cluster is received on the endpoint ``1``.
         The command's argument can be used to specify the duration of the effect.

      Button 0:
         .. include:: /includes/matter_sample_button.txt

      Button 1:
         Changes the light bulb state to the opposite one.
```

**Review by annwoj — APPROVED** [review]

**Pull request merged**